foreman/foreman.spec

```diff
@@ -577,6 +594,8 @@ sed -i 's/:locations_enabled: false/:locations_enabled: true/' config/settings.y
 sed -i 's/:organizations_enabled: false/:organizations_enabled: true/' config/settings.yaml
 export BUNDLER_EXT_NOSTRICT=1
 export BUNDLER_EXT_GROUPS="default assets"
+ln -s /usr/lib/node_modules/ .
```

Comment:
exporting NODE_PATH=%{node_sitedir} (and add node-packaging) on the line below might be preferable?